packages/@react-native-windows/cli/src/runWindows/runWindows.ts

```diff
@@ -55,12 +55,12 @@ function getPkgVersion(pkgName: string): string {
 }
 
 enum RunWindowsPhase {
-  None,
-  AutoLink,
-  FindBuildTools,
-  NuGetRestore,
-  FindSolution,
-  Deploy,
+  None = 0,
+  AutoLink = 1,
+  FindBuildTools = 2,
+  // NuGetRestore = 3, (Removed)
+  FindSolution = 4,
+  Deploy = 5,
```

Comment:
nit: this should be ok to not number since we are always logging the name of the enum value

Reply:
Oh, I see now.

If we're using the string value directly, this might be worth representing the type as a union type instead of enum. It's a sharp edge, but enums are somewhat discouraged in TS, since they don't have an ES6 equivalent.

So like, idiomatic TS if we wanted a string enum would be something like:

```ts
type RunWindowsPhase =
| 'None'
| 'AutoLink'
...
```